lib/oneshot_coverage.rb

```diff
@@ -22,17 +22,19 @@ def call(env)
   OneshotLog = Struct.new(:path, :md5_hash, :lines)
 
   class Reporter
-    def initialize(target_path:, logger:, emit_term: nil)
+    def initialize(target_path:, logger:, emit_term: nil, check_bundle_path:)
       @target_path = target_path
       @logger = logger
       @emit_term = emit_term
       if @emit_term
         @next_emit_time = Time.now.to_i + rand(@emit_term)
       end
 
+
       if defined?(Bundler)
         @bundler_path = Bundler.bundle_path.to_s
       end
+      @check_bundle_path = check_bundle_path
```

Comment:
Do we assign `@check_bundle_path` unless Bundler exist? 🤔

Reply:
Ah, `@check_bundle_path = check_bundle_path` should be limited to the case of defined Bundler.